### DISCUSSION TITLE
Worth it? Still active?

### DISCUSSION THREAD

**Discussion by Deenosaurus** [yusifsalam/t490-macos#49] [Q&A]
Hey there, I got a spare T490 which I would like to hackintosh. Is this guide still active and the way to follow? I can find just a little few information regarding to my lenovo laptop and hope this is the guide to follow.
I would love to run big sur on it.
Thanks for your answer @yusifsalam

**Comment by yusifsalam** [comment]
I don't use my t490 anymore so this repo won't be actively updated anymore (and hasn't been for a while), at least by me. This also means that I won't be providing any support or troubleshooting in case any issues may arise. I don't know if there are any other repositories for this laptop, at least I haven't seen any, but I also don't follow this topic actively anymore. 
With all that said, for Big Sur you should be fine with this, but you might run into issues if you try to install Monterey and later since I haven't tested it. If you do follow my repo, I recommend you update the kexts to the latest versions, the ones included here are pretty old. 
As for whether it's worth it, that depends. In my opinion, this was worth it before Apple silicon, nowadays not so sure about that. If you have a spare t490, I'd say macOS provides better overall experience than windows or linux, so might be worth a try. If you already have an m1 macbook or can get one, it's a significantly better laptop in pretty much every way possible and I wouldn't bother with hackintoshing. Personally I use a 2020 m1 air and haven't really touched the lenovo in almost a year.

**Reply by Deenosaurus** [reply]
Wow, I didn't really thought of getting an actual answer, thank you.

As I said I got a spare T490 laying around and I would love to tinker in macOS, as I never have touched this OS, but using an iPhone/iPad/Watch... I ain't planning on using it productively, since I have too much running on a windows machine and my work enviroment is almost microsoft only based, but I was always interessted in macOS.

Never got in touch with hackintoshing before and I can really understand, why you dont recommend using it since apple silicion. At first when intel and apple split up everyone thought "oh noes" but that was the best decision ever made. Now with these cpu/gpus, perfectly optimized for their used hardware and software, they can unleashed unbelievable power.

As for now, thanks and I will go through your repo, trying to find updated kexts (currently im clueless but I'll see). Have a great week.
